omnibus/config/projects/supermarket.rb

```diff
@@ -34,7 +34,7 @@
 override :git, version: "2.2.1"
 override :'chef-gem', version: '12.13.37'
 override :redis, version: '2.8.21'
-override :openssl, version: '1.0.2h'
+override :openssl, version: '1.0.2i'
```

Comment:
As noted by @nellshamrell elsewhere, this is still `i` despite bringing in the omnibus-software that provides `j`.

![gif-keyboard-16743417473528928748](https://cloud.githubusercontent.com/assets/517302/18883133/0ddd962a-84b0-11e6-9848-c652bde8560e.gif)